sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/TextAnalyticsRestClient.cs

```diff
@@ -20,22 +20,29 @@ namespace Azure.AI.TextAnalytics
     internal partial class TextAnalyticsRestClient
     {
         private string endpoint;
+        private string apiVersion;
         private ClientDiagnostics _clientDiagnostics;
         private HttpPipeline _pipeline;
 
         /// <summary> Initializes a new instance of TextAnalyticsRestClient. </summary>
         /// <param name="clientDiagnostics"> The handler for diagnostic messaging in the client. </param>
         /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
         /// <param name="endpoint"> Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com). </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> is null. </exception>
-        public TextAnalyticsRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint)
+        /// <param name="apiVersion"> Text Analytics API version (for example: v3.0). </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="apiVersion"/> is null. </exception>
+        public TextAnalyticsRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint, string apiVersion = "v3.1")
```

Comment:
it seems like all the docstrings say service version V3_1_Preview_5 but we are targeting "v3.1" here. Just checking

Reply:
No worries.
This is the default value in case no version is passed. in the `TextAnalyticsClient` I am always passing the version either specified by the user or the latest one: https://github.com/Azure/azure-sdk-for-net/blob/7df94b4303ff1a30c974a81328480f632a659012/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs#L69

https://github.com/Azure/azure-sdk-for-net/blob/7df94b4303ff1a30c974a81328480f632a659012/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs#L66